tests/std/tests/GH_002030_asan_annotate_string/env.lst

```diff
@@ -8,50 +8,30 @@ RUNALL_INCLUDE ..\prefix.lst
 RUNALL_CROSSLIST
 PM_CL="/Zi /wd4611 /w14640 /Zc:threadSafeInit-" PM_LINK="/debug"
 RUNALL_CROSSLIST
-PM_CL="-fsanitize=address /BE /c /EHsc /MD /std:c++14"
-PM_CL="-fsanitize=address /BE /c /EHsc /MDd /std:c++17 /permissive-"
-PM_CL="-fsanitize=address /BE /c /EHsc /MT /std:c++20 /permissive-"
-PM_CL="-fsanitize=address /BE /c /EHsc /MTd /std:c++latest /permissive-"
-PM_CL="-fsanitize=address /EHsc /MD /std:c++14"
-PM_CL="-fsanitize=address /EHsc /MD /std:c++17"
-PM_CL="-fsanitize=address /EHsc /MD /std:c++20"
-PM_CL="-fsanitize=address /EHsc /MD /std:c++latest /permissive- /Zc:char8_t- /Zc:preprocessor"
-PM_CL="-fsanitize=address /EHsc /MD /std:c++latest /permissive- /Zc:noexceptTypes-"
-PM_CL="-fsanitize=address /EHsc /MDd /std:c++14 /fp:except /Zc:preprocessor"
-PM_CL="-fsanitize=address /EHsc /MDd /std:c++17 /permissive-"
-PM_CL="-fsanitize=address /EHsc /MDd /std:c++20 /permissive-"
-PM_CL="-fsanitize=address /EHsc /MDd /std:c++latest /permissive- /Zc:wchar_t-"
-PM_CL="-fsanitize=address /EHsc /MDd /std:c++latest /permissive-"
-PM_CL="-fsanitize=address /EHsc /MT /std:c++latest /permissive- /analyze:only /analyze:autolog-"
-PM_CL="-fsanitize=address /EHsc /MT /std:c++latest /permissive-"
-PM_CL="-fsanitize=address /EHsc /MTd /std:c++latest /permissive"
-PM_CL="-fsanitize=address /EHsc /MTd /std:c++latest /permissive- /analyze:only /analyze:autolog-"
-PM_CL="-fsanitize=address /EHsc /MTd /std:c++latest /permissive- /fp:strict"
-PM_CL="-fsanitize=address /EHsc /MTd /std:c++latest /permissive-"
-PM_CL="/D_ANNOTATE_STRING /BE /c /EHsc /MD /std:c++14"
-PM_CL="/D_ANNOTATE_STRING /BE /c /EHsc /MDd /std:c++17 /permissive-"
-PM_CL="/D_ANNOTATE_STRING /BE /c /EHsc /MT /std:c++20 /permissive-"
-PM_CL="/D_ANNOTATE_STRING /BE /c /EHsc /MTd /std:c++latest /permissive-"
-PM_CL="/D_ANNOTATE_STRING /EHsc /MD /std:c++14"
-PM_CL="/D_ANNOTATE_STRING /EHsc /MD /std:c++14 /Zc:char8_t"
-PM_CL="/D_ANNOTATE_STRING /EHsc /MD /std:c++17"
-PM_CL="/D_ANNOTATE_STRING /EHsc /MD /std:c++17 /Zc:char8_t"

```

Comment:
You are losing these 14/17 with `/Zc:char8_t` configurations, due to pre-existing divergence between `-fsanitize=address` and `/D_ANNOTATE_STRING`. Intentional?

Reply:
Not intentional. I should have known better than to trust the description.